oc-chef-pedant/spec/api/oc_id_spec.rb

```diff
@@ -2,6 +2,38 @@
 require 'openssl'
 
 describe "oc_id API", :oc_id do
+
+  let(:request_headers) do
+    {
+      "Content-Type" => "application/x-www-form-urlencoded",
+      "Accept" => "text/html",
+      "Cookie" => csrf[:cookie]
+    }
+  end
+
+  let(:username) {  "pedant-oc-test-user-#{rand(10**7...10**8).to_s}" }
+  let(:password) { "foobar" } # hardcoded at the platform layer
+  let(:user_overrides) do
+    {
+      :overrides => {
+        "first_name" => "#{username}-first-name",
+        "middle_name" => "#{username}-middle-name",
+        "last_name" => "#{username}-last-name",
+        "display_name" => "#{username}-display-name",
+        "email" => "#{username}-email@host.com",
+      }
+    }
+  end
+
+  # create user before each test so that we don't regen the keys
+  # of the standard user used in other tests.
+  let(:oc_id_user) { platform.create_user(username, user_overrides) }
```

Comment:
That works for now. `shared` is a pedant thing that's hiding somewhere in our libs.  Semantically similar to `let` but less robust for overriding iirc, and it persists across examples so that they can be accessed for setup in before/after :all blocks.